client/mass/charts.js

```diff
@@ -237,7 +237,7 @@ function setRenderers(self) {
 					// summary/survival/cuminc all expect config.term{} to be a termsetting object, but not term (which is confusing)
 					// thus convert term into a termwrapper (termsetting obj)
 					// tw.q{} is missing and will be fill in with default settings
-					const tw = { id: term.id, term }
+					const tw = term.q ? term : { id: term.id, term }
```

Comment:
@airenzp it may be better to check for `term.term ? term : ...` instead of `term.q ? term : ...`, since a termwrapper is guaranteed to have {term}, but `q` may be filled-in only later by `fillTermWrapper()` or `fillTW()`.